release.groovy

```diff
@@ -2,8 +2,6 @@
 def ci (){
     stage('build'){
         sh 'npm install'
-        sh 'npm run build'
```

Comment:
Why are you removing this build step from the CI pipeline?  I added this in my commit that fixed this repo yesterday https://github.com/fabric8-ui/ngx-widgets/pull/97/commits/a994f48ee49baf2a06825c1ca4082b7e1f46123e.  Removing this step means folks can make changes that can break the release and it wont get caught by CI.  Also @joshuawilson why are you self merging PRs that relate to the pipelines? I've seen a few comments from you explaining the importance of reviews something I dont believe is always needed but in this case where you're making changed to the pipeline, something that you're not that familiar with I suggest you request and wait for a review before merging.

Reply:
Yesterday you told me to try to fix it on my own. Since I don't understand what is broken, I'm doing the best I can.

Reply:
Yesterday I said I'd fixed it https://github.com/openshiftio/openshift.io/issues/947#issuecomment-331159390 along with a link to the fix which I also commented on to say this fixes CI https://github.com/fabric8-ui/ngx-widgets/pull/97#issuecomment-331175860